src/context/trace_context.rs

```diff
@@ -25,11 +25,42 @@ use std::sync::Arc;
 
 use super::system_time::UnixTimeStampFetcher;
 
+/// Span is a concept that represents trace information for a single RPC.
+/// The Rust SDK supports Entry Span to represent inbound to a service
+/// and Exit Span to represent outbound from a service.
+///
+/// # Example
```

Comment:
This should have a copy on the readme.md in the root, right?

Reply:
Yes. fixed.